### ISSUE TITLE
[a11y] Cannot run WB Label

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1655]
![image](https://user-images.githubusercontent.com/37256050/169085148-0ccaece5-ba47-4242-89cc-d4ddf94eb3d8.png)

When trying to run WB Label in the `kufish` db, I get this error

**Comment by maxxxxxdlp** [comment]
You mean a WB data set called "Label"?

**Comment by grantfitzsimmons** [comment]
I thought that would be from Specify 6
